cmd/site-controller/deploy-watch-all-ns.yaml

```diff
@@ -0,0 +1,103 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: skupper-site-controller
+  namespace: default
```

Comment:
I wonder if we should create and use a specific namespace, e.g. 'skupper-site-controller' perhaps, instead of default? Obviously this is only an example, so it is not hugely important, but I think it might be a bit nicer than polluting default?